WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsInsightsDetailsViewModel.swift

```diff
@@ -107,6 +107,8 @@ class SiteStatsInsightsDetailsViewModel: Observable {
                 }
                 insightsReceipt = insightsStore.query(storeQuery)
 
+                let date = selectedDate ?? StatsDataHelper.currentDateForSite()
```

Comment:
This change may not be needed. 

With or without this change "Comments Total" details appear in the same way, no matter if I switch between Days/Weeks/Months/Years.

Reply:
Ditto - thanks for looking at this with a fresh set of eyes! I agree as well that I don't think this is needed.  I've removed